client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/tasks.tsx

```diff
@@ -1,23 +1,87 @@
+import { LaunchpadFlowTaskList } from './types';
+
 export const tasks = [
 	{
 		id: 'setup_newsletter',
```

Comment:
We're trying to be mindful of divergences from the existing checklist API, so that, if we end up needing to adopt the existing checklist API implementation, it'll be an easier refactor.

Do you happen to know if the data structure of this checklist somewhat matches the data structure of, say, the My Home checklist for example?

Reply:
@jeyip - Our current checklist implementation is similar to the `My Home` checklist.
`My Home` is using the follow data structure:

> id: string;
> isCompleted: boolean;
> taskType: 'user' | 'blog';